src/runtime/providers/unsplash.ts

```diff
@@ -8,7 +8,8 @@ const unsplashCDN = 'https://images.unsplash.com/'
 
 export const getImage: ProviderGetImage = (src, { modifiers = {}, baseURL = unsplashCDN } = {}) => {
   const operations = operationsGenerator(modifiers)
+  const hasQueryParams = src.includes('?')
   return {
-    url: withBase(joinURL(src + (operations ? ('?' + operations) : '')), baseURL)
+    url: withBase(joinURL(src + (operations ? ((hasQueryParams ? '&' : '?') + operations) : '')), baseURL)
```

Comment:
Tip: We can use `withQuery` utility from `ufo` to do this

Reply:
Hi @pi0, thank you for your feedback. I edited my changes to make use of `withQuery`.
I hope it is not a problem, but I had to use `getQuery('?' + operations)` to parse the operations, as `withBase` requires an object. 

I observed a plus side from using `withQuery`, now any pre-existent query parameters, like width, gets overwritten by the value set in the operations. Furthermore, I edited the playground example to reflect this.

Reply:
Nice workaround with getQuery 👍🏼